go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/UpCloudLtd/terraform-provider-upcloud
 go 1.17
 
 require (
-	github.com/UpCloudLtd/upcloud-go-api/v4 v4.8.0
+	github.com/UpCloudLtd/upcloud-go-api/v4 v4.8.1-0.20220906085109-8b75f46bdfc9
```

Comment:
Need to update this reference after https://github.com/UpCloudLtd/upcloud-go-api/pull/173 has been merged.

Reply:
Now referring to https://github.com/UpCloudLtd/upcloud-go-api/commit/5f6e629cfe2d383b102d1b5d941266335e120547